internal/core/src/index/CMakeLists.txt

```diff
@@ -14,15 +14,9 @@ set(INDEX_FILES
         Utils.cpp
         VectorMemIndex.cpp
         IndexFactory.cpp
+        VectorDiskIndex.cpp
```

Comment:
Can DISK_ANN be compiled on a Mac?

Reply:
No, libaio can not be install in mac. For this commit, `VectorDiskIndex.cpp` will process all vector index create/serialize/deserialize from file, not only DISKANN.